<launch>

    <node name="drive_inside_action_server" pkg="evo_autonom_driving" type="evo_drive_inside_server" output="screen" ns="evo_autonom_driving">
        <rosparam command="load" file="$(dirname)/../config/config.yaml" />
    </node>
</launch>